plugins/flytekit-ray/flytekitplugins/ray/models.py

```diff
@@ -148,13 +152,22 @@ def worker_group_spec(self) -> typing.List[WorkerGroupSpec]:
         """
         return self._worker_group_spec
 
+    @property
+    def enable_in_tree_autoscaling(self) -> bool:
```

Comment:
qq: why `in_tree`? Does Ray have other autoscaling strategies?

Reply:
Only have one autoscaling strategy. I have changed name to `enable_autoscaling`. Thank you!